packages/extensions/openapi-to-cadl/src/transforms/transform-object.ts

```diff
@@ -25,7 +25,7 @@ import { transformValue } from "../utils/values";
 
 const cadlTypes = new Map<SchemaType, string>([
   [SchemaType.Date, "plainDate"],
-  [SchemaType.DateTime, "zonedDateTime"],
+  [SchemaType.DateTime, "offsetDateTime"],
```

Comment:
```suggestion
  [SchemaType.DateTime, "offsetDateTime"],
```

so I think we want to recommend people to use `utcDateTIme`. Even though this change is probably safer maybe we should still try to use `utcDateTime` with a todo to go double check?

Reply:
I agree with Tim's idea. Since service teams will typically use utcDateTime, we should use that and let them check it if they need to update it.

Reply:
Makes sense